### DISCUSSION TITLE
关于文档的问题 | 星光

### DISCUSSION THREAD

**Discussion by giscus** [starguang/myblog#17] [Announcement]
# 关于文档的问题 | 星光

有这样一句话——程序员最讨厌两件事：别人不看文档和看文档。

https://www.xingguang.host/docs/%E9%9A%8F%E7%AC%94/%E5%85%B3%E4%BA%8E%E6%96%87%E6%A1%A3%E7%9A%84%E9%97%AE%E9%A2%98/

**Comment by lshcool** [comment]
那么本站生成文档的框架是哪个呢？
看起来还挺简洁的。
谢谢告知。

**Reply by 596094101** [reply]
是用这个生成的:docusaurus，也可以进行二次开发
